src/model/database.py

```diff
@@ -20,10 +34,25 @@ def serialize(self):
         self.__serialize_tables__()
 
     @staticmethod
-    def __validate__(schema_data):
-        if len(schema_data[Keys.DATABASE]) == 0 or schema_data[Keys.DATABASE].isspace():
-            raise WrongParameterError("No database detected")
+    def __validate__(schema_data, database_name):
+        if schema_data is None or schema_data[Keys.DATABASE].isspace():
+            if database_name is None or str(database_name).isspace():
+                raise WrongParameterError("No database detected")
 
     def __serialize_tables__(self):
-        for table in self.tables:
-            table.serialize()
+        for table_name in self.tables:
+            self.tables[table_name].serialize()
+
+    def get_table(self, table_name):
+        if table_name not in self.tables:
+            raise WrongParameterError("Wrong table entered")
+        return self.tables[table_name]
+
+    def set(self, table_name, data):
+        if table_name is not None:
```

Comment:
Table. Name can’t be null, it should throw an exception in validate in constructor if none